docs/ha_multi_dc.rst

```diff
@@ -0,0 +1,53 @@
+.. _ha_multi_dc:
+
+=================
+HA multi datacenter
+=================
+
+The high availability of a PostgreSQL cluster deployed in multiple data centers is based on replication, which can be synchronous or asynchronous (`replication_modes <replication_modes.rst>`_).
+
+In both cases, it is important to be clear about the following concepts:
+
+- Postgres can run as primary or standby leader only when it owns the leading key and can update the leading key.
+- You should run the odd number of etcd, ZooKeeper or Consul nodes: 3 or 5!
+
+Synchronous Replication
+----------------------------
+
+To have a multi DC cluster that can automatically tolerate a zone drop, a minimum of 3 is required.
+
+The architecture diagram would be the following:
+
+.. image:: _static/multi-dc-synchronous-replication.png
+
+We must deploy a cluster of etcd, ZooKeeper or Consul through the different DC, with a minimum of 3 nodes, one in each zone.
+
+Regarding postgres, we must deploy at least 2 nodes, in different DC. Then you have to set ``synchronous_mode: true`` in the global configuration (``patronictl edit-config``).
+
+This enables sync replication and the primary node will choose one of the nodes as synchronous.
+
+Streaming Replication (asynchronous)
```

Comment:
From this title it looks like only async replication is streaming. Can we just have 'Asynchronous Replication' here?

Reply:
for sure! i have already corrected the title, thanks for the feedback